app/views/layouts/application.html.haml

```diff
@@ -59,7 +59,9 @@
               Support links
             %ul{ role: 'navigation'} 
               %li
-                Having problems using this service? Email <a href="mailto:dss@crowncommercial.gov.uk">dss@crowncommercial.gov.uk</a> for support.
+                Having problems using this service? Email
```

Comment:
You're missing the full-stop at the end of the sentence after this change.

We'd usually do this:

```haml
= succeed '.' do
  Having problems with this service? Email
  = mail_to(support_email_address)
  for support
```